src/backend/tests/change-requests.test.ts

```diff
@@ -24,39 +25,81 @@ describe('Change Requests', () => {
   describe('reviewChangeRequest', () => {
     test('reviewer not found errors', async () => {
       jest.spyOn(prisma.user, 'findUnique').mockResolvedValue(null);
-      const response = await request(app).post('/review').send(whipPayloadObject);
-
-      expect(response.body).toStrictEqual({ message: `User with id #${whipPayloadObject.reviewerId} not found` });
-      expect(response.status).toBe(404);
+      try {
+        await ChangeRequestService.reviewChangeRequest(
+          whipPayloadObject.reviewerId,
+          whipPayloadObject.crId,
+          whipPayloadObject.reviewNotes,
+          whipPayloadObject.accepted,
+          '1'
+        );
+      } catch (e) {
+        // eslint-disable-next-line jest/no-conditional-expect
+        expect(e).toBeInstanceOf(NotFoundException);
```

Comment:
instead of a try catch, you can do:
```
await expect(() =>
        ChangeRequestService.reviewChangeRequest(
          whipPayloadObject.reviewerId,
          whipPayloadObject.crId,
          whipPayloadObject.reviewNotes,
          whipPayloadObject.accepted,
          '1'
        )
      ).rejects.toThrow(new NotFoundException('User', 1));
```

Reply:
You have no idea how long I looked for something like that. I had .toThrow, I didn’t know you had to put .rejects first omg